profile/README.md

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+    <img src="/assets/readme-banner.png" width="418" height="139" alt="" />
+</p>
+
+---
+
+## Cpp Linter
+
+**Linter your C-like code in workflow.** Including code formatting, diagnosing, and fixing typical programming errors.
```

Comment:
clang tools primarily support the C family (C/C++/Objective-C/Java). Other languages are C-like (eg. dart), but are not directly supported by clang. I also added some English corrections.
```suggestion
**Lint your C/C++ code in an automated workflow** including analysis regarding code formatting, diagnosing, and fixing typical programming errors.
```